language-server/modules/langserver-core/src/test/resources/completion/statement_context/source/wait_action_ctx_source16.bal

```diff
@@ -0,0 +1,25 @@
+import ballerina/module1;
+
+public function main() {
+
+    worker WA returns int|error {
+        return 0;
+    }
+  
+    worker WB returns string|error {
+        return "";
+    }
+    
+    future<int> fint = start getInt();
+    future<string> fstr = start getString(); 
+
+    map<string|error> myVar = wait {fieldName: module1: };
```

Comment:
Can we expect qualified name references for workers? IMO, no. Only usage will be like this:
```
map<string|error> myVar = wait {fieldName: start module1:<function_name>() };
```
Therefore, I think we don't need the qualified name reference part in the completion context (or return an empty list in that case). WDYT?

Reply:
Agreed about the usage. Returned an empty list instead.